virtual_ecosystem/models/hydrology/hydrology_model.py

```diff
@@ -440,11 +488,61 @@ def _update(self, time_index: int, **kwargs: Any) -> None:
                 veg_density_param=self.model_constants.veg_density_param,
             )
 
-            # TODO add canopy evaporation
+            # Calculate psychrometric constant
```

Comment:
psychrometric constant does not change throughout the month and can therefore move to the hydro-input or just outside the loop to declutter this function